docs/sources/flow/getting-started/collect-prometheus-metrics.md

```diff
@@ -0,0 +1,117 @@
+
+## Before you begin
+
+* Ensure that you basic familiarity with instrumenting applications with
+  Prometheus.
+* Have a set of Prometheus exports or applications exposing Prometheus metrics
+  that you want to collect metrics from.
+* Identify where you will write collected metrics, such as Prometheus, Grafana
+  Mimir, Grafana Cloud, or Grafana Enterprise Metrics.
+* Be familiar with the concept of [Components][] in Grafana Agent Flow.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+
+## Configure metrics delivery
+
+Before components can collect Prometheus metrics, you must have a component
+responsible for writing those metrics somewhere.
+
+The [prometheus.remote_write][] component is responsible for delivering
+Prometheus metrics to one or Prometheus-compatible endpoints. Once a
+`prometheus.remote_write` component is defined, other Grafana Agent Flow
+components can be used to forward metrics to it.
+
+To configure a `prometheus.remote_write` component for metrics delivery,
+complete the following steps:
+
+1. Add the following `prometheus.remote_write` component to your configuration file:
+
+   ```river
+   prometheus.remote_write "LABEL" {
+     endpoint {
+       url = "PROMETHEUS_URL"
+     }
+   }
+   ```
+
+2. Replace `LABEL` with a label to use for the component, such as `default`.
+   The label chosen must be unique across all `prometheus.remote_write`
+   components in the same configuration file.
+
+3. Replace `PROMETHEUS_URL` with the full URL of the Prometheus-compatible
+   endpoint where metrics will be sent, such as
+   `https://prometheus-us-central1.grafana.net/api/prom/push`.
+
+4. If your endpoint requires basic authentication, complete the following:
+
+    1. Paste the following inside of the `endpoint` block:
+
+       ```river
+       basic_auth {
+         username      = "USERNAME"
+         password_file = "PASSWORD_FILE"
+       }
+       ```
+
+    2. Replace `USERNAME` with the basic authentication username to use.
+
+    3. Replace `PASSWORD_FILE` with a path to a file containing the basic
+       authentication password or API key.
+
+5. If you have more than one endpoint to write metrics to, repeat the
+   `endpoint` block for additional endpoints.
+
+The following example demonstrates configuring `prometheus.remote_write` with
+multiple endpoints and mixed usage of basic authentication, and a
+`prometheus.scrape` component which forwards metrics to it:
+
+```river
+prometheus.remote_write "default" {
+  endpoint {
+    url = "http://localhost:9090/api/prom/push"
+  }
+
+  endpoint {
+    url = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+    basic_auth {
+      username      = env("REMOTE_WRITE_USERNAME")
+      password_file = "/etc/secrets/api-key"
+    }
+  }
+}
+
+prometheus.scrape "example" {
+  targets = [{
+    __address__ = "my-application:80",
+  }]
+
+  forward_to = [prometheus.remote_write.default.receiver]
+}
+```
+
```

Comment:
I don't want to make this any more difficult to follow, it's pretty succinct as-is. But do we want a sentence guiding our user to the Flow UI so they can see how data Flows, any debug info, etc?

Reply:
I don't think so, since we'd probably end up adding it to every single topic (users aren't expected to follow topics in any particular order). But if they follow the first topic, that does point them in the direction of the UI.

Reply:
Just an idea: there could be a "further reading" or "related pages" at the end which list the next things users could be interested in.

Reply:
We can consider doing that after we've fleshed out the topics more, but I'm also worried that we'll forget to maintain them if we try to keep a list of related topics at the bottom.